visualization/app/codeCharta/ui/areaSettingsPanel/areaSettingsPanel.component.html

```diff
@@ -26,7 +26,12 @@
 </md-slider-container>
 
 <md-input-container class="md-block">
-	<md-checkbox class="md-primary" ng-model="$ctrl._viewModel.dynamicMargin" ng-change="$ctrl.applyDynamicMargin()">
+	<md-checkbox
+		class="md-primary"
+		ng-model="$ctrl._viewModel.dynamicMargin"
+		ng-change="$ctrl.applyDynamicMargin()"
+		title="Use calculated margin for this map"
+	>
 		Dynamic Margin
 	</md-checkbox>
 </md-input-container>
```

Comment:
Below is the reset button. Those should also receive a tooltip for all different metrics.